client/wordpress-com.js

```diff
@@ -198,121 +191,110 @@ const sections = [
 	}
 ];
 
-editorPaths = [ '/post', '/page' ];
-if ( config.isEnabled( 'manage/custom-post-types' ) ) {
-	editorPaths.push( '/edit' );
-}
-
 sections.push( {
 	name: 'post-editor',
-	paths: editorPaths,
+	paths: [ '/post', '/page', '/edit' ],
```

Comment:
>Since manage/custom-post-types is true in every environment configuration I think that modifying this should produce no changes

Noticing I hadn't replied here. Yes, this should be fine, but we should probably also remove the config flag altogether if we're starting to make assumptions here that it's enabled in all environments.